### DISCUSSION TITLE
ASS Layers Switcher

### DISCUSSION THREAD

**Discussion by rRobis** [SubtitleEdit/subtitleedit#8253] [Ideas]
Currently, the subtitle list and waveform shows all the layers at once. It would be very nice if there would be a layer switcher, that would show only the subtitles in the current layer.

For example, if you switch to layer 1, subtitle list and waveform would only show the subtitles on layer 1, so that the background boxes and other layers wouldn't interrupt with the editing of the layer that I am currently on.

I think it would be much cleaner and less buggy than it is now.

Extra features:

- Show all layers position (i guess this is a must-have)
- Show multiple selected layers (from already used layers)
- ?

**Comment by rRobis** [comment]
@niksedk Any possibility that this could be implemented? I miss this feature till this day.

**Comment by midiminuit** [comment]
I'd really appreciate this too!
